woocommerce/woocommerce-ios
# Update prologue text and onboarding task

Closes: #11408 


## Description

This PR adds minor updates to the prologue screen and onboarding task list. Specifically:
- Fix typo for the text on prologue screen to state the 14-day free trial.
- Removed the badge text for Add your first product task list following design update in IpVQ5OMMkktucysPCWr9pK-fi-877-5942

## Testing instructions

- Log out of the app if needed.
- Confirm the prologue screen displays the correct text.
- Log in to a store with store onboarding tasks available.
- Notice that the Add your first product item no longer has a badge.

## Screenshots

| Prologue | Onboarding tasks | 
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/99a1a366-8870-4b9e-8073-f5e211c6d9d8" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/58eee13e-5de1-44b3-ad45-6a49911bd76a" width=320 /> |

---
